src/samples/article-poa/v1/complete.json

```diff
@@ -16,8 +16,8 @@
     },
     "pdf": "https://elifesciences.org/content/5/e14107.pdf",
     "subjects": [
-        "biochemistry",
-        "biophysics-structural-biology"
+        "bioch",
```

Comment:
Mainly to avoid delimiting problems if we use them in URIs and use a hyphen to separate it from a slug. Though we could always use a slash instead.

(The original subject IDs here I just made up based on the name, not sure if that's a good idea in case they get renamed, though I'm not sure if we would ever rename them. Plus we probably wouldn't use a subject ID + slug combination...)